mpv-plugin/open-in-mpv/open-in-mpv-2.4.1.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit go-module desktop xdg-utils
+
+DESCRIPTION="Simple host script (in Go) for simple web extension to open videos in mpv"
+HOMEPAGE="https://github.com/Baldomo/open-in-mpv https://addons.mozilla.org/en-US/firefox/addon/iina-open-in-mpv"
+SRC_URI="https://github.com/Baldomo/open-in-mpv/releases/download/v${PV}/open-in-mpv_${PV}.tar.gz"
+S="${WORKDIR}/${PN}_${PV}"
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	media-video/mpv
+"
+
+src_compile() {
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 ego build -ldflags="-s -w -X main.Version=${PV}" -o ${PN} ./cmd/open-in-mpv/
```

Comment:
This is optional: if it can be compiled on other architectures, like aarch/arm64, you can condition amd64 here. Otherwise, it's fine.

Reply:
I believe that it should work 99%, but in the perfect world scenario somebody needs to check on real hardware...

Reply:
> I believe that it should work 99%, but in the perfect world scenario somebody needs to check on real hardware...

It's ok, you can keep it like this, and if someone manifests interest, they will ask for additional architectures.

Reply:
> I don't understand why it is hardcoded to Linux and amd64, this should be set automatically by go eclass. If there is no specific reason, I think it should be removed.

I was thinking more from the conditional pov like a IF it's amd64?, but GOOS and GOARCH can indeed go away from the arguments if no conditionals are specified.
I don't think there is a specific reason to keep them, go does build based on arch detection, so it's perfectly fine without them :+1: 
CC @vitaly-zdanevich 

I'll check tomorrow as well, to maje sure everything is fine on my side as well.